x-pack/plugins/session_view/common/types/process_tree/index.ts

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export enum EventKind {
+  event = 'event',
+  signal = 'signal',
+}
+
+export enum EventAction {
+  fork = 'fork',
+  exec = 'exec',
+  end = 'end',
+  output = 'output',
+}
```

Comment:
```suggestion
export const enum EventKind {
  event = 'event',
  signal = 'signal',
}

export const enum EventAction {
  fork = 'fork',
  exec = 'exec',
  end = 'end',
  output = 'output',
}
```

💅  [`const enum NAME`](https://www.typescriptlang.org/docs/handbook/enums.html#const-enums) will inline the strings at compile time. That way this file is not actually generating runtime code.

Reply:
Some codebases are against using `export const enum`, thank you @janmonschke for letting us know we can use it here